tests/pipeline_operations_test.py

```diff
@@ -1,14 +1,39 @@
 import unittest
 import pyspark
 
+from pipeline_dp import DataExtractors
 from pipeline_dp.pipeline_operations import SparkRDDOperations
 from pipeline_dp.pipeline_operations import LocalPipelineOperations
+from pipeline_dp.pipeline_operations import BeamOperations
 
 
 class PipelineOperationsTest(unittest.TestCase):
     pass
 
 
+class BeamOperationsTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+      cls.ops = BeamOperations()
+      cls.data_extractors = DataExtractors(
+          partition_extractor=lambda x: x[1],
+          privacy_id_extractor=lambda x: x[0],
+          value_extractor=lambda x: x[2])
+
+    def test_filter_partitions_noop(self):
+      col = [(1, 6, 1), (2, 7, 1), (3, 6, 1), (4, 7, 1), (5, 8, 1)]
+      public_partitions = []
+      result = self.ops.filter_partitions(col, public_partitions, self.data_extractors, "Public partition filtering")
+      assert result == col
+
+
+    def test_filter_partitions_remove(self):
```

Comment:
could you please also when public_partition is PCollection?

Here is the how to test Beam pipeline https://beam.apache.org/documentation/pipelines/test-your-pipeline/